packages/core-modules/contracts/modules/UpgradeModule.sol

```diff
@@ -1,32 +1,15 @@
 //SPDX-License-Identifier: MIT
 pragma solidity ^0.8.0;
 
-import "@synthetixio/core-contracts/contracts/proxy/UniversalProxyImplementation.sol";
+import "@synthetixio/core-contracts/contracts/proxy/UUPSImplementation.sol";
 import "../mixins/OwnerMixin.sol";
-import "../storage/ProxyStorage.sol";
 
-contract UpgradeModule is UniversalProxyImplementation, ProxyStorage, OwnerMixin {
+contract UpgradeModule is UUPSImplementation, OwnerMixin {
     function upgradeTo(address newImplementation) public override onlyOwner {
         _upgradeTo(newImplementation);
     }
 
     function getImplementation() public view returns (address) {
```

Comment:
Should the public getImplementation() be in the UUPSImplementation (available for all contracts extending it or not)? I opted for not, but I'm open to suggestions

Reply:
I like the "not" approach until necessary 👮

Reply:
Your call. Sounds good.